docs/libraries/http/reference/data-types.md

```diff
@@ -268,6 +421,11 @@ model TypeSpec.Http.PlainData<Data>
 | ---- | -------------------------------------- |
 | Data | The model to spread as the plain data. |
 
+#### Properties
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
```

Comment:
If there are no properties, should we omit the section or render something like 
```suggestion
#### Properties

None
```

Reply:
yeah I was thinking about that but also felt like a model without properties was an edge case. Don't have a strong opinion on this.

Reply:
Agree it's an edge case, but I was reading docs and saw a model with a blank properties table, I would assume it's a bug and open an issue rather than "Oh yeah, that means it doesn't have any properties.". Elsewhere in the doc rendering code, we check that lengths are >0 before rendering a table.